src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { TablePeople } from './TablePeople';
+import { Person } from '../types/Person';
+import { Loader } from './Loader';
+
+type Props = {
+  people: Person[],
+  isLoading: boolean,
+  isError: boolean,
+};
+
+export const PeoplePage: React.FC<Props> = ({
+  isLoading, isError, people,
+}) => {
+  const { slug = '' } = useParams();
+
+  return (
+    <>
```

Comment:
`<div className="container">` 
You have a container to avoid using react fragments

Reply:
Same